<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>319</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Module</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbSinReset">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Note</string>
    </property>
   </widget>
   <widget class="QSlider" name="vsPitch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vsAttack">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vsDecay">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decay</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPitch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>440</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbAttack">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.10</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDecay">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.20</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbWave">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tri</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="vsTime">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="sliderPosition">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbAutoTime">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbAutoTime_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAutoPlay">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wave Shape</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
